### DISCUSSION TITLE
What is DNS propagation?

### DISCUSSION THREAD

**Discussion by Roadman95** [community/community#52124] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Like the title says

**Comment by MustafaBaban** [comment, marked as answer]
DNS propagation refers to the process by which changes to a domain's DNS (Domain Name System) records are spread or propagated throughout the internet. When a user types a domain name into their browser, the browser looks up the domain's IP address by querying DNS servers. When a change is made to a domain's DNS records, it can take some time for all the DNS servers around the world to update their records with the new information. This process can take anywhere from a few minutes to several hours, depending on factors such as the TTL (Time To Live) value set in the DNS record and the number of DNS servers that need to update their records. During the propagation period, some users may see the old DNS records while others see the new records, leading to temporary inconsistencies in the availability of the domain

**Reply by MargolisLarson** [reply]
why is my new site not coming up? MargolisLarson It has been two days

**Reply by MustafaBaban** [reply]
Two days is a long time, try to check if you set up everything correctly.

**Comment by MargolisLarson** [comment]
We already did that. Can you look at it for me? Please!

From: Mustafa ***@***.***>
Date: Friday, June 28, 2024 at 12:21 PM
To: community/community ***@***.***>
Cc: Tod Margolis ***@***.***>, Comment ***@***.***>
Subject: Re: [community/community] What is DNS propagation? (Discussion #52124)

Two days is a long time, try to check if you set up everything correctly.

**Reply by MustafaBaban** [reply]
Show your configuration, and il take a look at it

**Comment by MargolisLarson** [comment]
##- Please type your reply above this line -##

Your request (376894) has been updated. To add additional comments, reply to this email.

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 28, 2024, 12:00 EDT

Hi Tod,

I just received your message.

You can let them know that you have set up the domain to point to a page but the domain seems to be connected to a different page.

In this case, the issue looks like it's on their end as the domain is not connecting to your funnel, and when using third-party tools we see the domain is still broadcasting the old settings as we see multiple A records associated with different IP addresses to then one we set up for Funnel Pages.

Please let me know if this makes sense or if you have any other questions.


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylive

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 28, 2024, 11:35 EDT

Hello Tod,

Thanks for your response.

Looking at the DNS settings, the connection is set but the domain is not resolving to your funnel but a different page:

[IcfmyijaJWLw4nV6IaPpY1Ox4]

The following image is from who.is which is an external service that's not affiliated with us in any way and is designed to read DNS records:

[raJxMR4BEbn79v1qCx5jNbsUB]

As you can see, your domain is still broadcasting the old settings.

Here is an image from another DNS lookup tool called https://dnschecker.org/ showing this domain connected to other IP addresses than the FP one:

[MCmBmYJmf9Uj3UYzkyXfOXVZI]

Keep in mind that we were able to confirm that Netlify is the Cpanel and is where the settings need to be according to these name servers that are also showing on who.is:

[fjXBbGAQnMUwEzw2P5CuUCk7g]

This seems to be an issue with the DNS and I would recommend reaching out to Netlify to check if there are any issues and so they can confirm how long can the DNS changes take to propagate.

Please let me know if you have any questions or if we can assist you with anything else!


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylive

[default-avatar-80.png]

Tod M

Jun 28, 2024, 11:14 EDT

Hey MargolisLarson!

A sign in attempt requires further verification because we did not recognize your device. To complete the sign in, enter the verification code on the unrecognized device.

Device: Chrome on Windows
Verification code: 760862

If you did not attempt to sign in to your account, your password may be compromised. Visit https://github.com/settings/security to create a new, strong password for your GitHub account.

If you'd like to automatically verify devices in the future, consider enabling two-factor authentication on your account. Visit https://docs.github.com/articles/configuring-two-factor-authentication to learn about two-factor authentication.

If you decide to enable two-factor authentication, ensure you retain access to one or more account recovery methods. See https://docs.github.com/articles/configuring-two-factor-authentication-recovery-methods in the GitHub Help.

Thanks,
The GitHub Team
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 28, 2024, at 10:08 AM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 28, 2024, 11:08 EDT

Hi Tod,

My apologies, it seems that this code expired.

Please share the new one sent to your email so I can access the dashboard.


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylive

[default-avatar-80.png]

Tod M

Jun 28, 2024, 11:01 EDT

Hey MargolisLarson!

A sign in attempt requires further verification because we did not recognize your device. To complete the sign in, enter the verification code on the unrecognized device.

Device: Chrome on Windows
Verification code: 368620

If you did not attempt to sign in to your account, your password may be compromised. Visit https://github.com/settings/security to create a new, strong password for your GitHub account.

If you'd like to automatically verify devices in the future, consider enabling two-factor authentication on your account. Visit https://docs.github.com/articles/configuring-two-factor-authentication to learn about two-factor authentication.

If you decide to enable two-factor authentication, ensure you retain access to one or more account recovery methods. See https://docs.github.com/articles/configuring-two-factor-authentication-recovery-methods in the GitHub Help.

Thanks,
The GitHub Team Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 28, 2024, at 9:48 AM, Tod Margolis ***@***.******@***.***>> wrote:

﻿ Ok, let me look
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 28, 2024, at 9:20 AM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Tod M

Jun 28, 2024, 10:59 EDT

That way you needed
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 28, 2024, at 9:48 AM, Tod Margolis ***@***.******@***.***>> wrote:

﻿ Ok, let me look
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 28, 2024, at 9:20 AM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Tod M

Jun 28, 2024, 10:49 EDT

Hey MargolisLarson!

A sign in attempt requires further verification because we did not recognize your device. To complete the sign in, enter the verification code on the unrecognized device.

Device: Chrome on Windows
Verification code: 368620

If you did not attempt to sign in to your account, your password may be compromised. Visit https://github.com/settings/security to create a new, strong password for your GitHub account.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 28, 2024, at 9:48 AM, Tod Margolis ***@***.******@***.***>> wrote:

﻿ Ok, let me look
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 28, 2024, at 9:20 AM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Tod M

Jun 28, 2024, 10:48 EDT

Ok, let me look
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 28, 2024, at 9:20 AM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 28, 2024, 10:20 EDT

Hi Tod,

Happy Friday!

Please share the GitHub device verification code sent to your email so I can access the dashboard.

Looking forward to your response.


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylive

[default-avatar-80.png]

Tod M

Jun 28, 2024, 08:57 EDT

Good morning Dominic,
The site is not propagated yet. Is there a way you can Double check it?
Or do you need me to do that? I’m on the road today, but I can get off the road with a little bit of notice.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 26, 2024, at 8:33 AM, Tod Margolis ***@***.******@***.***>> wrote:

﻿
I am here

[default-avatar-80.png]

Tod M

Jun 26, 2024, 09:33 EDT

I am here

[default-avatar-80.png]

Tod M

Jun 25, 2024, 18:01 EDT

We have access to everything now. Please call me so we can get this done. Please call me today…

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 25, 2024, 17:40 EDT

Hi Tod,

Thanks for your response.

The changes can only be done from the Cpanel, we need to create an A record that points to Funnel Pages instead of the ones I shared that are pointing to a different IP addreess.

What we need is access to the Cpanel or if your client can add the A record and delete the old ones.

Looking forward to your update!


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylive

[default-avatar-80.png]

Tod M

Jun 25, 2024, 17:36 EDT

He said it’s not a sea panel
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 4:35 PM, Tod Margolis ***@***.******@***.***>> wrote:

﻿
<image0.png>

Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 4:33 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

Attachment(s)
image0.png<https://localgenius.zendesk.com/attachments/token/dYQoRIcOYU1EuHcwrfaxOQfmW/?name=image0.png>

[default-avatar-80.png]

Tod M

Jun 25, 2024, 17:36 EDT

[image0.png]
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 4:33 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

Attachment(s)
image0.png<https://localgenius.zendesk.com/attachments/token/5LLoV63hgTiQ93lQzm6vw4OhF/?name=image0.png>

[default-avatar-80.png]

Tod M

Jun 25, 2024, 17:36 EDT

I confirmed the email with them
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 4:33 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 25, 2024, 17:32 EDT

Hi Tod,

Thanks for youwr response.

What we need is access to the domain provider's Cpanel so we can make the DNS changes.

Can you please confirm if GiftHub is where your Cpanel resides?

Looking forward to your update.


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylive





[default-avatar-80.png]

Tod M

Jun 25, 2024, 16:45 EDT

Netlify doesn’t use cPanel specifically, but you should have access to everything you need to manage/transfer the site. You’re an owner for the site now. Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 3:22 PM, Tod Margolis ***@***.******@***.***>> wrote:

﻿ that’s what they told me.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 2:40 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Tod M

Jun 25, 2024, 16:41 EDT

From greg

No, Netlify is where you should find everything you need for the current site. You should have an invitation from Netlify to join my team so that you can manage the site there.

GitHub is just where the code for the site is stored, and I use my GitHub credentials to log into Netlify, which is why I can’t share the cress.

Tod Margolis

Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 3:22 PM, Tod Margolis ***@***.******@***.***>> wrote:

﻿ that’s what they told me.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 2:40 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Tod M

Jun 25, 2024, 16:35 EDT

are we hosting it on David server where the website is?
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 3:22 PM, Tod Margolis ***@***.******@***.***>> wrote:

﻿ that’s what they told me.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 2:40 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Tod M

Jun 25, 2024, 16:23 EDT

so where are we? I don’t understand what you said.. are you able to get this up and working today?
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 3:22 PM, Tod Margolis ***@***.******@***.***>> wrote:

﻿ that’s what they told me.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 2:40 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Tod M

Jun 25, 2024, 16:22 EDT

that’s what they told me.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 25, 2024, at 2:40 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 25, 2024, 15:40 EDT

Hey Tod,

It is the first time I have heard about a domain hosted on GitHub, are you sure this is where the CPanel resides?

Remember we found regarding this:

[SXRXsLaESeVzVPemTi3zFVFXQ]

Remember the A record we need is 54.84.101.21.


Warm Regards,

Dominic S.
Customer Support Guru


P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.

Register Now: http://localgenius.com/joinfridaylive

[default-avatar-80.png]

Tod M

Jun 24, 2024, 16:36 EDT

https://app.screencast.com/ru1QXO9dQkBfd

https://github.com<https://github.com/>

Dentonhand.com

User Name is: MargolisLarson

Password is Boston09$$&

Video show the same information

Please let me know how you make out.

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 24, 2024, 12:54 EDT

Hi Tod,

Thanks for your response.

I will be reaching out in a moment once I have the chance.

Let us know if you need anything else in the meantime!


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylives

[default-avatar-80.png]

Tod M

Jun 21, 2024, 21:41 EDT

I have what I need so lets set a time on Monday if that’s works for you?

Tod

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 11, 2024, 18:08 EDT

Hello Tod,

Thanks for the update.

Please use this link to rebook our session:

https://calendly.com/d/ck8b-gcd-mfg/funnels

Let me know if we can assist you with anything in the meantime!


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylive

[default-avatar-80.png]

Tod M

Jun 11, 2024, 17:13 EDT

I need to reschedule today’s appointment. I’m sorry.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 6, 2024, at 7:16 PM, Tod Margolis ***@***.******@***.***>> wrote:

﻿ Let me know when you have 10 or 15 minutes on Friday tomorrow and I’ll take my laptop with me. if you can text me, that’s great if not, send me an email and I’ll try to check my email every hour. Just give me a 30 minute heads up.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 6, 2024, at 1:37 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Tod M

Jun 7, 2024, 13:19 EDT

Target CPAnal
Domain company user and password

Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 6, 2024, at 7:16 PM, Tod Margolis ***@***.******@***.***>> wrote:

﻿ Let me know when you have 10 or 15 minutes on Friday tomorrow and I’ll take my laptop with me. if you can text me, that’s great if not, send me an email and I’ll try to check my email every hour. Just give me a 30 minute heads up.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 6, 2024, at 1:37 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Tod M

Jun 6, 2024, 20:17 EDT

Let me know when you have 10 or 15 minutes on Friday tomorrow and I’ll take my laptop with me. if you can text me, that’s great if not, send me an email and I’ll try to check my email every hour. Just give me a 30 minute heads up.
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 6, 2024, at 1:37 PM, Dominic (Strategic Marketer) ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 6, 2024, 14:37 EDT

Hi Tod,

Thanks for getting back tot me.

I will let you know of any openings if one becomes available.

Let us know if you need anything else in the meantime!


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylive

[default-avatar-80.png]

Tod M

Jun 6, 2024, 13:32 EDT

I made a time if anything comes up sooner please let me know
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 6, 2024, at 11:01 AM, Strategic Marketer ***@***.******@***.***>> wrote:

﻿

[default-avatar-80.png]

Dominic (Strategic Marketer)

Jun 5, 2024, 11:01 EDT

Hi Tod,

Happy Wednesday!

Let's discuss this over a quick Zoom call.

Please use this link to book a session with me:

https://calendly.com/d/cp5q-yb4-r7k/funnels

Looking forward to hearing more about this.


Warm Regards,

Dominic S.
Customer Support Guru

P.S. Are you ready to discover something new? Join us every week on Friday Live for training on strategy, products and get your questions answered LIVE by the product team.


Register Now: http://strategicmarketer.com/joinfridaylive

[default-avatar-80.png]

Tod M

Jun 5, 2024, 09:28 EDT

Hello Dominick,
Dr. Larson is just about ready to add him to your host to bring his site live. I know his URL it is DentonHand.com What do I need from him? I know who his domain is from. This is the info from the current webmaster ie the Dr. Larson brother. below is a email from Dr L brother.
I just invited you as domain manager on Squarespace for the dentonhand.com domain. Tim is the owner of the domain, so he can likely login to Squarespace to manage the domain as well.

Once you confirm you can access it, I'll add you to the others that we own as well.

As far as hosting the actual site, it's hosted on Netlify as a Hugo website. I assume you're rebuilding the site entirely, but if you need access to the current site, just let me know.

Ok, What is next step?

This email is a service from Strategic Marketer. Delivered by Zendesk<https://www.zendesk.com/support/?utm_campaign=text&utm_content=Strategic+Marketer&utm_medium=poweredbyzendesk&utm_source=email-notification>
[7W0R1P-9Y4XJ]
Tod Margolis
Margolis Photography
3336 Brittany Drive
Flower Mound, TX 75022
401.258.7619 cell
972.539.9125
Sent from my iPhone

On Jun 28, 2024, at 12:41 PM, Mustafa ***@***.***> wrote:

﻿

Show your configuration, and il take a look at it
